Description: Fix the import path for fuzzy-patricia
 Debian package golang-github-ozeidan-fuzzy-patricia uses import path
 "github.com/ozeidan/fuzzy-patricia/patricia".
Author: Jongmin Kim <jmkim@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/pkg/gui/controllers/helpers/suggestions_helper.go
+++ b/pkg/gui/controllers/helpers/suggestions_helper.go
@@ -12,7 +12,7 @@
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/samber/lo"
 	"golang.org/x/exp/slices"
-	"gopkg.in/ozeidan/fuzzy-patricia.v3/patricia"
+	"github.com/ozeidan/fuzzy-patricia/patricia"
 )
 
 // Thinking out loud: I'm typically a staunch advocate of organising code by feature rather than type,
--- a/pkg/gui/gui.go
+++ b/pkg/gui/gui.go
@@ -49,7 +49,7 @@
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/samber/lo"
 	"github.com/sasha-s/go-deadlock"
-	"gopkg.in/ozeidan/fuzzy-patricia.v3/patricia"
+	"github.com/ozeidan/fuzzy-patricia/patricia"
 )
 
 const StartupPopupVersion = 5
--- a/pkg/gui/types/common.go
+++ b/pkg/gui/types/common.go
@@ -11,7 +11,7 @@
 	"github.com/jesseduffield/lazygit/pkg/tasks"
 	"github.com/jesseduffield/lazygit/pkg/utils"
 	"github.com/sasha-s/go-deadlock"
-	"gopkg.in/ozeidan/fuzzy-patricia.v3/patricia"
+	"github.com/ozeidan/fuzzy-patricia/patricia"
 )
 
 type HelperCommon struct {
